### PR TITLE
support -r in requirement files

### PR DESCRIPTION
**Pull request by coobas**

### CODE REVIEW

File: scurvysin/__init__.py [hunk 1]
```diff
@@ -64,6 +65,12 @@ def expfl(self) -> list:
             return ['install']
 
 
+class PiPArgParser(argparse.ArgumentParser):
```
Comment (janpipek) [inline scurvysin/__init__.py]:
```suggestion
class PipArgumentParser(argparse.ArgumentParser):
    """Parser to support additional pip --arguments"""
```

File: scurvysin/__init__.py [hunk 2]
```diff
@@ -110,7 +117,11 @@ def parse_requirements_file(path: str) -> List[str]:
         for line in infile:
             line = line.split("#", maxsplit=1)[0].strip()
             if line.startswith("-"):
-                raise NotImplementedError("Pip flags in requirements files are not understood.")
+                try:
+                    pip_args = PiPArgParser().parse_args(line.split())
```
Comment (janpipek) [inline scurvysin/__init__.py]:
```suggestion
                    pip_args = PipArgumentParser().parse_args(line.split())
```

File: scurvysin/__init__.py [hunk 3]
```diff
@@ -64,6 +65,12 @@ def expfl(self) -> list:
             return ['install']
 
 
+class PiPArgParser(argparse.ArgumentParser):
+    def __init__(self):
+        super().__init__()
+        self.add_argument("-r", dest="requirement_file", help="Path to pip requirements file.")
```
Comment (janpipek) [inline scurvysin/__init__.py]: Shouldn't we support `--requirement` too?

Comment (coobas) [reply scurvysin/__init__.py]: Definitely, I'll add it

Comment (coobas) [reply scurvysin/__init__.py]: Added, without unit tests so far, would add after #14

**Review by janpipek — APPROVED** [review]

**Pull request merged**